scripts/fauxalysis_from_docking.py

```diff
@@ -1,37 +1,234 @@
+    in_dir, out_dir, best_structure_dict, frag_dir=None, cmpd_to_frag_dict=None
+):
+    """
+    Convert original-style output structure to a fragalysis-style output
+    structure.
+
+    Parameters
+    ----------
+    in_dir : str
+        Top-level directory of original-style output
+    out_dir : str
+        Top-level directory of new output
+    frag_dir : str
+        Path to fragalysis results directory.
+    best_structure_dict : dict
+        Of the form {('Compound_ID', dimer): 'Structure_Source'} where "dimer" is a boolean.
+    """
+    os.makedirs(f"{out_dir}", exist_ok=True)
+
+    ## Create list of sdf files we will make so that we can concatenate them at the end
+    cmpd_sdf_list = []
+
+    ## Loop through dict and parse input files into output files
+    for (compound_id, dimer), best_str in best_structure_dict.items():
+        ## Make sure input exists
+        dimer_s = "dimer" if dimer else "monomer"
+        compound_in_dir = f"{in_dir}/{compound_id}/{dimer_s}/{best_str}"
+        compound_out_dir = f"{out_dir}/{compound_id}"
+
+        ## If inputs don't exist, else if the output directory already exists, don't waste time
+        if not check_output(compound_in_dir):
+            print(
+                (
+                    f"No results found for {compound_id}/{dimer_s}/{best_str}, "
+                    "skipping"
+                ),
+                flush=True,
+            )
+            continue
+        else:
+            print(
+                (
+                    f"Generating fauxalysis...\n"
+                    f"\tfrom: {compound_in_dir}\n"
+                    f"\tto: {compound_out_dir}"
+                ),
+                flush=True,
+            )
+
+        ## Create output directory if necessary
+        os.makedirs(compound_out_dir, exist_ok=True)
+
+        ## Load necessary files
+        du = oechem.OEDesignUnit()
+        oechem.OEReadDesignUnit(f"{compound_in_dir}/predocked.oedu", du)
+        prot = oechem.OEGraphMol()
+        du.GetProtein(prot)
+        lig = load_openeye_sdf(f"{compound_in_dir}/docked.sdf")
+
+        ## Set ligand title, useful for the combined sdf file
+        lig.SetTitle(f"{compound_id}_{best_str}")
+
+        ## Give ligand atoms their own chain "L" and set the resname to "LIG"
+        for atom in lig.GetAtoms():
+            residue = oechem.OEAtomGetResidue(atom)
+            residue.SetChainID("L")
+            residue.SetName("LIG")
+            residue.SetHetAtom(True)
+            oechem.OEAtomSetResidue(atom, residue)
+
+        ## First save apo
+        save_openeye_pdb(prot, f"{compound_out_dir}/{best_str}_apo.pdb")
+
+        ## Combine protein and ligand and save
+        oechem.OEAddMols(prot, lig)
+        save_openeye_pdb(prot, f"{compound_out_dir}/{compound_id}_bound.pdb")
+
+        ## Remove Hs from lig and save SDF file
+        for a in lig.GetAtoms():
+            if a.GetAtomicNum() == 1:
+                lig.DeleteAtom(a)
```

Comment:
@kaminow I copied this over from your script, is there a reason you did this? Because I think this is what is causing John's error here:
https://asapdiscovery.slack.com/archives/C03QD9UM9TP/p1665812798018309

Reply:
oh interesting, yeah I did it because the SDF files in fragalysis don't have Hs in them, but there's probably a better way of filtering atoms to write out than just deleting them

Reply:
in this case I think the fragalysis SDFs are wrong and we should write out Hs